.github/workflows/auto-label-issues.yml

```diff
@@ -0,0 +1,36 @@
+name: Auto Label Issues Based on Title
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  label-issues:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Auto-label issues based on title
+        run: |
+          ISSUE_TITLE=$(jq -r .issue.title "$GITHUB_EVENT_PATH" | tr '[:upper:]' '[:lower:]')
+
+          if [[ "$ISSUE_TITLE" == *"bug"* ]]; then
```

Comment:
@guruashish-dev  Could you check the labels support in this repo. And add those labels too. Thanks